backend/src/agents/retriever_tools.py

```diff
@@ -144,148 +147,137 @@ def initialize(
         """
-        if RetrieverAgent.general_retriever is None:
+        if RetrieverTools.general_retriever is None:
             raise ValueError('General Retriever not initialized')
 
-        docs = RetrieverAgent.general_retriever.invoke(input=query)
+        docs = RetrieverTools.general_retriever.invoke(input=query)
         return format_docs(docs)
 
     @staticmethod
     @tool
     def retrieve_cmds(query: str) -> Tuple[str, list[str], list[str]]:
         """
-        Retrieve information on the commands and tools available in the OpenROAD project and OpenROAD-Flow-Scripts. \
-        This includes descriptions, usage guidelines, command syntax, examples, and best practices.
-        The tool assists users by delivering clear, accurate, and relevant details to help them effectively utilize the following commands and tools within OpenROAD and OpenROAD-Flow-Scripts:
- 
-        Here's a list of all the tools available:
-        ANT: Antenna Rule Checker
-        CTS: Clock Tree Synthesis
-        DFT: Design For Testing
-        DPL: Detailed Placement
-        DRT: Detailed Routing
-        FIN: Metal Fill
-        GPL: Global Floorplanning
-        GPL: Global Placement
-        GRT: Global Routing
-        GUI: Graphical User Interface
-        IFP: Initialize Floorplan
-        MPL: Macro Placement
-        MPL: Hierarchical Macro Placement
-        ODB: OpenDB
-        Chip-level Connections
-        PAD: Pad
-        PAR: Partition Manager
-        PDN: Power Distribution Network
-        PPL: Pin Placement
-        PSM: IR Drop Analysis
-        RSX: Parasitics Extraction
-        RMP: Restructure
-        RSZ: Gate Resizer
-        STT: Rectilinear Steiner Tree
-        TAP: TapCell
-        UPF: Read Unified Power Format
-        UTL: Utilities
-
-        The tools can be represented by their abbreviations as well: ANT,CTS,DFT,DPL,DRT,FIN,GPL,GRT,GUI,IFP,MPL,ODB,PAD,PAR,PDN,PPL,PSM,RSX,RMP,RSZ,STT,TAP,UPF,UTL     
-        
+        Retrieve information on the commands available in the OpenROAD project and OpenROAD-Flow-Scripts.\
+        This includes descriptions, usage guidelines, command syntax, examples, and best practices.\
+        This tool assists users by delivering clear, accurate, and\
+        relevant details to help them effectively utilize the following commands within OpenROAD and OpenROAD-Flow-Scripts:\
+        Here's a list of all the commands available:
+        ANT: Antenna Rule Checker\
+        CTS: Clock Tree Synthesis\
+        DFT: Design For Testing\
+        DPL: Detailed Placement\
+        DRT: Detailed Routing\
+        FIN: Metal Fill\
+        GPL: Global Floorplanning\
+        GPL: Global Placement\
+        GRT: Global Routing\
+        GUI: Graphical User Interface\
+        IFP: Initialize Floorplan\
+        MPL: Macro Placement\
+        MPL: Hierarchical Macro Placement\
+        ODB: OpenDB\
+        Chip-level Connections\
```

Comment:
this line is missing a corresponding acronym. intended?

Reply:
Yes, thanks for pointing it out. It's corrected now.